includes/misc-functions.php

```diff
@@ -1056,12 +1056,32 @@ function edd_is_promo_active() {
  * For existing installs, this option is added whenever the function is first used.
  *
  * @since 2.11.4
+ * @param bool $use_first_order Optional flag to use the first existing order date.
  * @return int The timestamp when EDD was activated.
  */
-function edd_get_activation_date() {
+function edd_get_activation_date( $use_first_order = false ) {
```

Comment:
Are there any circumstances you can think of where we WOULDN'T want to use the first order?

Reply:
Sure, during `edd_run_install`, I don't think we want to use the first order, since there won't be one. The query may not take much juice in that case, I guess, so maybe removing the parameter and check is okay....

Otherwise, if we are calling this function, then maybe we could invert this logic and set it to true here, but `false` during `edd_run_install`.

Reply:
Yeah this should be a pretty quick query, and should only ever run one time. So I don't think it will hurt if we run it unnecessarily on activation.

I'm fine with either:

1. Removing the parameter and always running this; or
2. Inverting the logic and defaulting to `true`.

Go with whichever one you prefer and I'll approve it. :grin:

Reply:
I'm good with removing the parameter--simplifies things and there won't be any questions in the future. And if the Queen of Queries is okay with it then I can't disagree. 👑